app/views/placements/support/providers/provider_options.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: new_placements_support_provider_path) %>
```

Comment:
I think we want to preserve the search term in the form -- that's what happens on the prototype. But if you've heard differently from Simon, just ignore this.

Reply:
As discussed, this is rather complicated due to handling the search params different for the js and non-js versions. Should be a separate ticket.

Reply:
Given how the JS autocomplete works this will need to be separated out into a more complex ticket.
Likely this will need event listeners added, etc.
https://trello.com/c/e5SwYKuK/138-fix-back-links-on-providers-schools-onboarding-pages-to-pre-fill-the-search-input-new-page-with-users-input